.eslintrc.js

```diff
@@ -0,0 +1,267 @@
+module.exports = {
```

Comment:
Weird config for eslint, if you downloaded airbnb config you are expected to use it, but I hope you got the idea how it works after our call

Reply:
Yes)